src/components/auth/PasswordReset.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import resetPassword from '../../actions/resetPassword';
+import './PasswordReset.scss';
+
+export class PasswordResetForm extends Component {
+  state = {
+    email: '',
+  };
+
+  handleSubmit = async (event) => {
+    event.preventDefault();
+    const { resetPassword: passwordReset, history } = this.props;
+    await passwordReset(this.state);
+    setTimeout(() => { history.push('/passwordresetmessage'); }, 3000);
```

Comment:
Can you explain the need for setTimeout here?

Reply:
I was using that to delay redirecting a user to a new page. But it has been removed.